### ISSUE TITLE
Switching to a data file

### ISSUE THREAD

**Issue by evankrocker** [rypan/jekyll-db#2]
It seems like it should be pretty easy to switch from posts to a data file.  I created a data file and then switch all the markup to pull from the data file but it borks the whole site.  Is it not possible?

**Comment by rypan** [comment]
You are right. It is totally possible now. When I first pulled this together, Github Pages didn't support data files. I should make an update to support both.

**Comment by evankrocker** [comment]
I went through a few of the files and made adjustments to point to a data file but I'm just getting build errors.  The build error emails are not even telling me where the error actually is.  Weird.

**Comment by sergiospagnuolo** [comment]
I've tried this myself, and it worked to display the data. However, making searches in the search input was not possible. Maybe was doing something wrong. Is there a way to make data files searchable?
